src/NzbDrone.Core/MediaFiles/RenameMovieFileService.cs

```diff
@@ -139,38 +139,27 @@ private void RenameFiles(List<MovieFile> movieFiles, Movie movie, string oldMovi
 		public void RenameMoviePath(Movie movie, bool shouldRenameFiles = true)
 		{
 			var newFolder = _filenameBuilder.BuildMoviePath(movie);
-	        if (newFolder != movie.Path && movie.PathState == MoviePathState.Dynamic)
-	        {
-
-	            if (!_configService.AutoRenameFolders)
-	            {
-	                _logger.Info("{0}'s movie should be {1} according to your naming config.", movie, newFolder);
-	                return;
-	            }
+            if (newFolder != movie.Path && movie.PathState == MoviePathState.Dynamic)
+            {
+                if (!_configService.AutoRenameFolders)
+                {
+                    _logger.Info("{0}'s movie should be {1} according to your naming config.", movie, newFolder);
+                    return;
+                }
 
-	             _logger.Info("{0}'s movie folder changed to: {1}", movie, newFolder);
+                _logger.Info("{0}'s movie folder changed to: {1}", movie, newFolder);
                 var oldFolder = movie.Path;
                 movie.Path = newFolder;
 
-	            _diskProvider.MoveFolder(oldFolder, movie.Path);
-
-				if (false)
-				{
-					var movieFiles = _mediaFileService.GetFilesByMovie(movie.Id);
-					_logger.ProgressInfo("Renaming movie files for {0}", movie.Title);
-					RenameFiles(movieFiles, movie, oldFolder);
-					_logger.ProgressInfo("All movie files renamed for {0}", movie.Title);
-				}
-
-				_movieService.UpdateMovie(movie);
-
-                if (_diskProvider.GetFiles(oldFolder, SearchOption.AllDirectories).Count() == 0)
-                {
-                    _recycleBinProvider.DeleteFolder(oldFolder);
-                }
+                _diskProvider.MoveFolder(oldFolder, movie.Path);
+            }
 
+			var movieFiles = _mediaFileService.GetFilesByMovie(movie.Id);
```

Comment:
This was intentionally disabled. Files shouldn't be renamed when you rename a folder. If you do it here, the files will be sent back to the old folder, since the movie hasn't been updated with the new path.

Reply:
What is the trigger to rename files then? 

In my tests it worked, because I on line 159 (my code) I did replace oldFolder by newFolder

Reply:
Any hints?